tests/std/tests/P0896R4_common_iterator_death/test.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <iterator>
+
+#include <test_death.hpp>
+using namespace std;
+
+struct simple_input_iter {
+    using value_type      = int;
+    using difference_type = int;
+
+    value_type operator*() const {
+        return 0;
+    }
+    value_type operator->() const {
+        return 0;
+    }
+    simple_input_iter& operator++() {
+        return *this;
+    }
+    simple_input_iter operator++(int) {
+        return *this;
+    }
+
+    bool operator==(simple_input_iter const&) const = default;
+    bool operator==(const default_sentinel_t&) const {

```

Comment:
Inconsistent east `const` vs. `const` west.

Reply:
Went with the only true const

Reply:
`const` west is `const` worst!